### DISCUSSION TITLE
Can't create project.

### DISCUSSION THREAD

**Discussion by AndrzejStopa** [AntlerVC/firetable#1018] [Support / Q&A] [answered]
I can't create a new project after deleting it.
![image](https://user-images.githubusercontent.com/117684785/208097752-f50d6044-9d7e-445f-b213-8b09dfd9c87c.png)

**Comment by bofeiw** [comment, marked as answer]
Hi @AndrzejStopa, we offer the first project free. After that, you will need to subscribe to Rowy Pro. Feel free to play around with it with a new account, that should allow you to create a free project.

**Reply by AndrzejStopa** [reply]
Ok, so there is not one free project, but the first one ;) That's what I did as you wrote :)

**Reply by bofeiw** [reply]
Hi @AndrzejStopa, we decided to give everyone 1 free project at all time. If you delete your first project, you can always create another one. Hope this helps
